BoardingPass/BoardingNavigationController.swift

```diff
@@ -91,43 +161,43 @@ public class BoardingNavigationController: UINavigationController {
      - parameter animated: Specify true to animate the transition or false
      if you do not want the transition to be animated.
      */
-    public func popToPreviousViewController(animated animated: Bool) {
-        if let poppableViewController = topViewController.flatMap(boardingInfoAfter) {
-            popToAndInsertIfNeeded(poppableViewController, animated: animated)
+    public func popToPreviousViewController(animated: Bool) {
+        if let poppableViewController = topViewController.flatMap({ boardingInfo(afterController: $0) }) {
+            popToAndInsertIfNeeded(controller: poppableViewController, animated: animated)
         }
     }
 }
 
-extension BoardingNavigationController: UINavigationControllerDelegate {
+// MARK: - Extensions
 
-    public func navigationController(navigationController: UINavigationController,
-                                     animationControllerForOperation operation: UINavigationControllerOperation,
-                                     fromViewController fromVC: UIViewController,
-                                     toViewController toVC: UIViewController) -> UIViewControllerAnimatedTransitioning? {
-        return animatedTransitioningProvider?(operation)
-    }
-
-    public func navigationController(navigationController: UINavigationController, interactionControllerForAnimationController animationController: UIViewControllerAnimatedTransitioning) -> UIViewControllerInteractiveTransitioning? {
-        return interactionController
-    }
-}
-
-// MARK: - Actions
+// MARK: Actions
 private extension BoardingNavigationController {
-    func popToAndInsertIfNeeded(viewController: UIViewController, animated: Bool) {
+    func popToAndInsertIfNeeded(controller viewController: UIViewController, animated: Bool) {
```

Comment:
#pp rename to `popToAndInsertIfNeeded(_ viewController: UIViewController, animated: Bool)`